mednafen/pce_fast/vdc.cpp

```diff
@@ -849,7 +843,11 @@ void VDC_RunFrame(EmulateSpecStruct *espec, bool IsHES)
          };
 
          DisplayRect->x = 0;
-         DisplayRect->w = ws[correct_aspect][vce.dot_clock];
+         if(hoverscan == 1 && vce.dot_clock == 1){
+			   DisplayRect->w = 352;
+		   }else{
+			   DisplayRect->w = ws[correct_aspect][vce.dot_clock];
```

Comment:
[That's this array here](https://github.com/westonlast/beetle-pce-fast-libretro/blob/master/mednafen/pce_fast/vdc.cpp#L852) where we probably just use the second line.
It could be removed and replaced by defined_width I linked above.